### ISSUE TITLE
Value of the pretty parameter

### ISSUE THREAD

**Issue by marcverhagen** [clamsproject/clams-python#41]
The mapping from parameter value to service behavior is not intuitive when pinging the service with curl. The current code just requires the `pretty` parameter to be there with a value that is not the empty string. Any of the following gives you a pretty print

```bash
$> curl -H "Accept: application/json" -X POST -d@example-mmif.json http://0.0.0.0:5000/?pretty=False
$> curl -H "Accept: application/json" -X POST -d@example-mmif.json http://0.0.0.0:5000/?pretty=tralala
$> curl -H "Accept: application/json" -X POST -d@example-mmif.json http://0.0.0.0:5000/?pretty=True
```

Whereas with the following you get no pretty print

```bash
$> curl -H "Accept: application/json" -X POST -d@example-mmif.json http://0.0.0.0:5000/
$> curl -H "Accept: application/json" -X POST -d@example-mmif.json http://0.0.0.0:5000/?pretty
```

I think it would be better if any value but 'False' or 'false' or '0' would result in a pretty print.

Somewhat related to this is that it would be nice for the `appmetadata()` to also accept a pretty parameter.

**Comment by marcverhagen** [comment]
There is some code on this in the 41-pretty branch: https://github.com/clamsproject/clams-python/commit/981e9a71

**Comment by keighrim** [comment]
aa9c3d6f9109c0e0c7f951fa5675ab533185ec89 is an attempt for refactoring "boolifying" method to the HTTP request handler class. It also leaves place holders for additional type conversions.

**Comment by keighrim** [comment]
Closed via #43 .
